packages/orchestration/src/utils/time.js

```diff
@@ -25,7 +24,10 @@ export function makeTimestampHelper(timer, timerBrand) {
      *   since unix epoch
      */
     async getTimeoutTimestampNS(relativeTime) {
-      const currentTime = await E(timer).getCurrentTimestamp();
+      const [currentTime, timerBrand] = await Promise.all([
+        E(timer).getCurrentTimestamp(),
+        brandCache || E(timer).getTimerBrand(),
+      ]);
       const timeout =
```

Comment:
@0xpatrickdev I managed to avoid passing `timerBrand` to every orchestration contract.